packages/paste-website/src/components/icons-list/IconTile.tsx

```diff
@@ -50,7 +50,7 @@ const IconTile = React.forwardRef<HTMLButtonElement, IconTileProps>(({children,
       {children}
       <Box as="div" display="flex" color="colorTextWeak" marginTop="space30" maxWidth="100%">
         <ScreenReaderOnly>Show details for:</ScreenReaderOnly>
-        <Truncate>{icon.name.replace('Icon', '')}</Truncate>
+        <Truncate title={icon.name}>{icon.name.replace('Icon', '')}</Truncate>
```

Comment:
Do we need to pass `icon.name` to the replace function in the `title` as well?

Reply:
Wasn't sure about that, thanks for starting a thread. It looks like `icon.name` always has an `Icon` suffix that is removed by the replace function (so I'm guessing "ChatIcon"). 
I used the full icon name thinking that including the suffix would give the title more descriptive clarity, since the goal of the ticket was to use the full text that is being truncated.
Is that repetitive or unnecessary in this case?

Reply:
> ...including the suffix would give the title more descriptive clarity...

That makes sense to me. Keeping Icon there is fine with me since that is the full name of each.